src/main/java/com/hcl/appscan/jenkins/plugin/scanners/DynamicAnalyzer.java

```diff
@@ -335,17 +413,44 @@ public FormValidation doCheckScanFile(@QueryParameter String scanFile) {
 			return FormValidation.ok();
 		}
 
-		public FormValidation doCheckTarget(@QueryParameter String target,@RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context, @QueryParameter String presenceId) {
+		public FormValidation doCheckTarget(@QueryParameter String target,@RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context, @QueryParameter String presenceId, @QueryParameter boolean rescanDast) {
```

Comment:
The logic in this method is extremely difficult to follow. There are numerous "if" blocks, may of which are doing the same checks and there is a nested if block within one of them, as well. This needs to be cleaned up or it will likely result in maintenance problems later.

Reply:
Okay, updated the logics in the method for better readability.